bootcamp/workshops/dbta_may_2024/1. RAG_basic.ipynb

```diff
@@ -28,7 +28,7 @@
   },
```

Comment:
<p>Why changing this file at all?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
This is a commit by a previous person